lightkurve/interact.py

```diff
@@ -551,7 +551,7 @@ def show_interact_widget(tpf, notebook_url='localhost:8888',
         aperture_mask = np.zeros(tpf.flux.shape[1:]).astype(bool)
         aperture_mask[0, 0] = True
 
-    lc.meta['aperture_mask'] = aperture_mask
+    lc.meta['APERTURE_MASK'] = aperture_mask
 
```

Comment:
Note: a small number of meta data keywords, such as `label` and `aperture_mask`, don't come from the FITS header but are instead set by lightkurve methods.  I wonder if we should adopt lowercase or uppercase for those.  I guess uppercase for everything makes things more consistent.

Reply:
I feel keeping all the meta keys uppercase would be easier for users.

In the current PR, it does let meta to have both uppercase and non-uppercase keys (it's a `dict`), a test is added to show how we resolve such cases if someone access it via attributes.

I considered going further, emitting a warning if someone sets meta with a a non uppercase keys, but it might be overly prescriptive (and extra codes needed to replace the standard `dict` with one that checks key cases).